source/bot.ts

```diff
@@ -39,8 +42,10 @@ bot.on("ready", async () =>
    setInterval(() => delayedTaskService.handleDueDelayedTasks(), 600000);
    setInterval(() => tumbleWeedService.handleTumbleWeed(bot), 600000);
    console.log("INFO: All services loaded. Bot is ready.")
-   
-   //Reimplement activity set. It isnt working rn anyways so I just kicked it out.
+
+   const fs = require('fs');
+   let config = JSON.parse(fs.readFileSync('botconfig.json', 'utf8'))
```

Comment:
this is obsolete since we already have `import botConfig from "./botconfig.json";` which works just fine if you just want to read the json

Reply:
You are supposed to use `botConfig.activity` from the already important botconfig

You dont need to do 
```
   const fs = require('fs');
   let config = JSON.parse(fs.readFileSync('botconfig.json', 'utf8'))
```